chaosinstana/probes.py

```diff
@@ -52,3 +108,27 @@ def convert_time(source_time: str) -> int:
             logger.error("Invalid date string provided")
             raise ActivityFailed("Invalid date string provided")
         return int(start_dt.timestamp() * 1000)
+
+
+def has_severity(event_data, severity_level):
```

Comment:
it would be good if the two parameters had a type annotation

Reply:
and the function a return type annotation, or

```
from typing import NoReturn

def .... -> NoReturn:
...

````

when a function does not return anything explicitely

Reply:
how do I declare a function that returns a boolean, cant seem to find a way the pleases the linter